ethportal-peertest/src/jsonrpc.rs

```diff
@@ -87,12 +87,18 @@ impl JsonRpcEndpoint {
             JsonRpcEndpoint {
                 method: "portal_statePing".to_string(),
                 id: 6,
-                params: Params::Array(vec![Value::String(buddy_node.clone())]),
+                params: Params::Array(vec![
+                    Value::String(buddy_node.clone()),
+                    Value::String("010100".to_string()),
```

Comment:
My understanding is that we do not implement `validate_endpoint_response` here because that will be taken up in #211. Is that correct?

Reply:
Yes, correct, it is just a placeholder in this PR.